content/resources/how-to-guides/state-changes-across-task-executions.md

```diff
@@ -0,0 +1,171 @@
+    | :------------------ | :---------------- | :------------ | -----: | :----- | :------------------- |
+    | example-measurement | example-tag-value | example-field |   30.0 | ok     | 2022-01-01T00:00:00Z |
+    | example-measurement | example-tag-value | example-field |   50.0 | crit   | 2022-01-01T00:01:00Z |
+
+
+4. Write “states” back to InfluxDB. You can write the data to a new measurement or to a new bucket. To write the data to a new measurement, use [`set()`](/flux/v0.x/stdlib/universe/set/) to update the value of the `_measurement` column in your “states” data. 
+
+    ```js
+    states
+        // (Optional) Change the measurement name to write the data to a new measurement
+        |> set(key: "_measurement", value: "new-measurement")
+        |> to(bucket : "example-bucket") 
+    ```
+
+5. Find the latest value from the previous task run and store it in a variable “last_state_previous_task”,
+
+    ```js
+    last_state_previous_task =
+        from(bucket: "example-bucket")
+            |> range(start: date.sub(d: task.every, from: tasks.lastSuccess(orTime: -task.every))
+            |> filter(fn: (r) => r._measurement == "example-measurement")
+            |> filter(fn: (r) => r.tagKey == "example-tag-value")
+            |> filter(fn: (r) => r._field == "example-field")
+            |> last() 
+    ```
+
+    Where `last_state_previous_task` might look like: 
+
+    | _measurement        | tagKey1           | _field        | _value | _level | _time                |
+    | :------------------ | :---------------- | :------------ | -----: | :----- | :------------------- |
+    | example-measurement | example-tag-value | example-field |   55.0 | crit   | 2021-12-31T23:59:00Z |
+
+6. Union “states” and “last_state_previous_task”. Store this data in a variable “unioned_states”.
+
+    ```js
+    unioned_states = union(tables: [states, last_state_previous_task])
+
+    // use the sort() function to guarantee that you preserve the time order
+
+    |> sort(columns: ["_time"], desc: true)
+    ```
+
+    Where `unioned_states` might look like: 
+
+    | _measurement        | tagKey1           | _field        | _value | _level | _time                |
+    | :------------------ | :---------------- | :------------ | -----: | :----- | :------------------- |
+    | example-measurement | example-tag-value | example-field |   55.0 | crit   | 2021-12-31T23:59:00Z |
+    | example-measurement | example-tag-value | example-field |   30.0 | ok     | 2022-01-01T00:00:00Z |
+    | example-measurement | example-tag-value | example-field |   50.0 | crit   | 2022-01-01T00:01:00Z |
+
+7. Use [`monitor.stateChangesOnly()`](/flux/v0.x/stdlib/influxdata/influxdb/monitor/statechangesonly/) to return only rows where the state changed in “unioned_states”. Store this data in a variable “state_changes”.
+
+    ```js
+    state_changes = unioned_states 
+        |> monitor.stateChangesOnly()
+    ```
```

Comment:
~~~suggestion
7. Use [`monitor.stateChangesOnly()`](/flux/v0.x/stdlib/influxdata/influxdb/monitor/statechangesonly/) to return only rows where the state changed in “unioned_states”. Store this data in a variable, “state_changes”.

    ```js
    state_changes =
        unioned_states 
            |> monitor.stateChangesOnly()
    ```
~~~